server/routes/index.ts

```diff
@@ -0,0 +1,335 @@
+          statusCode: error.statusCode,
+          body: error.message,
+        });
+      }
+    },
+  );
+
+  router.delete(
+    {
+      path: `${API_PREFIX}/configuration/{resourceName}/{id}`,
+      validate: {
+        params: schema.object({
+          resourceName: schema.string(),
+          id: schema.string(),
+        }),
+      }
+    },
+    async (context, request, response) => {
+      const client = securityConfigClient.asScoped(request);
+      let esResp;
+      try {
+        esResp = await client.callAsCurrentUser('opendistro_security.deleteResource', { resourceName: request.params.resourceName, id: request.params.id });
+        return response.ok({
+          body: {
+            message: esResp.message,
+          }
+        })
+      } catch (error) {
+        return response.custom({
+          statusCode: error.statusCode,
+          body: error.message,
+        });
+      }
+    }
+  );
+
+  router.post(
+    {
+      path: `${API_PREFIX}/configuration/{resourceName}`,
+      validate: {
+        params: schema.object({
+          resourceName: schema.string(),
+        }),
+        body: schema.any(),
+      },
+    },
+    async (context, request, response) => {
+      try {
+        validateRequestBody(request.params.resourceName, request.body)
+      } catch (error) {
+        return response.badRequest({ body: error });
+      }
+      const client = securityConfigClient.asScoped(request);
+      let esResp;
+      try {
+        esResp = await client.callAsCurrentUser('opendistro_security.saveResourceWithoutId', { resourceName: request.params.resourceName, body: request.body });
+        return response.ok({
+          body: {
+            message: esResp.message,
+          }
+        });
+      } catch (error) {
+        return response.custom({
+          statusCode: error.statusCode,
+          body: error.message,
+        });
+      }
+    }
+  );
+
+  router.post(
+    {
+      path: `${API_PREFIX}/configuration/{resourceName}/{id}`,
+      validate: {
+        params: schema.object({
+          resourceName: schema.string(),
+          id: schema.string(),
+        }),
+        body: schema.any(),
+      },
+    },
+    async (context, request, response) => {
+      try {
+        validateRequestBody(request.params.resourceName, request.body)
+      } catch (error) {
+        return response.badRequest({ body: error });
+      }
+      const client = securityConfigClient.asScoped(request);
+      let esResp;
+      try {
+        esResp = await client.callAsCurrentUser('opendistro_security.saveResource', { resourceName: request.params.resourceName, id: request.params.id, body: request.body });
+        return response.ok({
+          body: {
+            message: esResp.message,
+          }
+        });
+      } catch (error) {
+        return response.custom({
+          statusCode: error.statusCode,
+          body: error.message,
+        });
+      }
+    }
+  );
+  // test endpoint, DELETE it
```

Comment:
confused, does it refer to below api?  ${API_PREFIX}/configuration/validate

Reply:
good catch, I had deleted the testing API, but forgot to delete this comment